nelson/blueprints.go

```diff
@@ -0,0 +1,86 @@
+package nelson
```

Comment:
If anything here is taken from the Nelson CLI codebase it will need to have the license header with attribution in order to confirm with the license terms